app/src/main/java/com/rajatgoyal/puzzle15/data/GameContract.java

```diff
@@ -3,25 +3,27 @@
 import android.net.Uri;
 import android.provider.BaseColumns;
 
+import com.rajatgoyal.puzzle15.BuildConfig;
+
 /**
  * Created by rajat on 15/9/17.
  */
 
 public class GameContract {
```

Comment:
This couldv'e been an interface when all it has is `public static final` constants (interface stuff).